app/models/katello/concerns/setting_extensions.rb

```diff
@@ -6,8 +6,11 @@ module SettingExtensions
       included do
         validates :value, inclusion: { in: ::Katello::RootRepository::DOWNLOAD_POLICIES },
           if: ->(setting) { setting.name == 'default_download_policy' }
```

Comment:
This can also be rewritten using [setting validations DSL](https://github.com/theforeman/foreman/blob/develop/developer_docs/how_to_create_a_plugin.asciidoc#validate-setting-values), that will abstract the if condition for you.